src/main/java/com/amalitech/gpuconfigurator/model/Category.java

```diff
@@ -0,0 +1,35 @@
+package com.amalitech.gpuconfigurator.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity(name = "Category")
+@Table(name = "category")
+public class Category {
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private UUID id;
+
+    @Column(name = "category_name", nullable = false)
```

Comment:
This is just a suggestion for next time, but you could have just done `name` instead of `category_name`, The table name is `category`, it's easy to infer that the column with column name `name` means category name